### PR TITLE
Match _core & -addon changelog

### PR DESCRIPTION
**Pull request by raphmur**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -2,17 +2,25 @@
 
 ## 0.0.11
 
-- BREAKING CHANGES: Keep a backup of your VIN & MAC Addr
+### Breaking change & Upgrade Instruction
+- No need to provide MAC addresses anymore for presence detection
```
Comment (baylanger) [inline CHANGELOG.md:6]: Not true just yet. No need to provide MAC address IF the end user doesn't want to do presence detection.

File: CHANGELOG.md [hunk 2]
```diff
@@ -2,17 +2,25 @@
...
+- **BREAKING CHANGE - save config before update**: Now supports **list** of VINS. You will need to adjust configuration. Existing **entities** from v0.0.10f will not be affected.
```
Comment (baylanger) [inline CHANGELOG.md:7]: Do we want to specify `of VINS and MAC addrs` ?

File: CHANGELOG.md (hunk 1 above)
Comment (raphmur) [reply CHANGELOG.md:6]: Yes indeed

File: CHANGELOG.md (hunk 2 above)
Comment (raphmur) [reply CHANGELOG.md]: yes done

**Review by baylanger — APPROVED** [review]
Looks good to me!

**Pull request merged**